spec/routes/status_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Citygram::Routes::Status do
+  include Citygram::Routes::TestHelpers
+
+  describe 'GET /status' do
+    it 'returns a non-empty body' do
+        stub_request(:get, "https://api.twilio.com/2010-04-01").
```

Comment:
Could we abstract this to an environment variable? It's also used [here](https://github.com/codeforamerica/citygram/pull/194/files#diff-7c7d0096272b7f766c0b8676099c7b08R17)